jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -237,14 +245,44 @@ var module = module;
       };
       var body = {
         name: name,
-        controller: controller,
         templates: [baseTemplate]
       };
+      if (location) {
+        body.Location = location;
+      }
+      if (controller) {
+        body.controller = controller;
+      }
       var url = [this.url, 'model', ownerName].join('/');
       _makeRequest(this.bakery, url, 'POST', body, handler);
     },
 
     /**
+      List regions available in the given cloud.
+
+      @public listRegions
+      @param cloud {String} The name of the cloud (for instance "aws");
```

Comment:
s/;/. 

Could you also add that these values can be retrieved from the other api call.